### PR TITLE
Class notes Tuesday

### PR DESCRIPTION
**Pull request by hstone102**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +15,7 @@ <h1 class="kermit">Index of Class Notes</h1>
 
     <h2>Daily Notes</h2>
     <ul>
+        <li><a href="notes/2018-09-25.html">September 24, 2018</a></li>
```
Comment (bmizepatterson) [inline index.html]: Change date inside the <a> tag to September 25, 2018

Comment (hstone102) [reply index.html]: I think I fixed it.  Thank you!

**Comment by maymeclayton** [comment]
+1

**Comment by devinepk** [comment]
+1!!!

**Pull request merged**